pkg/operator/apiserver/audit/audit_policies.go

```diff
@@ -22,6 +22,21 @@ const (
 	auditPolicyAsset = "pkg/operator/apiserver/audit/manifests/audit-policies-cm.yaml"
 )
 
+// DefaultPolicy brings back the default.yaml audit policy to init the api
+func DefaultPolicy() ([]byte, error) {
+	cm, err := GetAuditPolicies("kube-apiserver-audit-policies", "openshift-kube-apiserver")
+
+		if err != nil {
+		return nil, fmt.Errorf("failed to retreive audit policies", err)
+	}
+	cmData, err := []byte(cm.Data["default.yaml"]), err
```

Comment:
is what you have equivalent to the snippets below?
```
defaultData, ok := cm.Data["default.yaml"]
if !ok || len(defaultData) == 0 {
  return nil, fmt.Errorf("failed to retrieve the default policy", err)
}
```

Reply:
Thanks for the rec here!